modules/javafx.graphics/src/main/native-glass/gtk/glass_window.cpp

```diff
@@ -775,6 +776,8 @@ WindowContextTop::WindowContextTop(jobject _jwindow, WindowContext* _owner, long
     if (frame_type == TITLED) {
         request_frame_extents();
     }
+
+    event_serial = 0;
```

Comment:
should we use GDK_CURRENT_TIME instead of 0 ?
GDK_CURRENT_TIME is defined also as 0 and represent current time.

Reply:
GDK_CURRENT_TIME is always  0, but it looks better. Changed it.